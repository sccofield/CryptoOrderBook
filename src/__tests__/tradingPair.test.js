import React from 'react';
import toJSON from 'enzyme-to-json';
import { shallow } from 'enzyme';


import TradingPair from '../components/tradingPair';

describe('Trading pair component', () => {
  const props = { 
    pairs: [],
    selectedPair: { },
    handleChange: jest.fn(() => Promise.resolve())
  }
  it('should render the component', () => {
    const wrapper = shallow(<TradingPair {...props} />);
    expect(wrapper.contains('Select Currency Pair')).toEqual(true);    
    expect(toJSON(wrapper)).toMatchSnapshot();
  });

});
