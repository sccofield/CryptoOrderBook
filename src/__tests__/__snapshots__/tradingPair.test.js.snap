// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trading pair component should render the component 1`] = `
<div
  className="trading-pair"
>
  <select
    className="form-control"
    id="inputState"
    onChange={[Function]}
  >
    <option>
      Select Currency Pair
    </option>
  </select>
  <div
    className="details"
  >
    <p>
      <span>
        Symbol: 
      </span>
    </p>
    <p>
      <span>
        Description: 
      </span>
    </p>
  </div>
</div>
`;
